storage/innobase/xtrabackup/src/xbcloud/s3.h

```diff
@@ -243,6 +243,8 @@ class S3_client {
 
   ulong get_max_backoff() { return max_backoff; }
 
+  void retry_error(Http_response *resp, bool *retry);
```

Comment:
where is the virtual declaration? I assume S3_client and swift client are derived classes?

Reply:
Client class doesn't extend any base class. S3_object_store and swift_object_store are the ones that are derived from Object_store class

Reply:
ok